server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanResourceDescriptionResolver.java

```diff
@@ -94,7 +95,11 @@ private String getVariableBundleKey(final String name, final String... variable)
     static {
         sharedAttributeResolver = new HashMap<String, String>();
         // shared cache attributes
+        /*for(AttributeDefinition attribute : CacheResource.CACHE_ATTRIBUTES) {
```

Comment:
Actually I want to remove the whole list of puts underneath and go with this dynamic approach